lib/pengine/utils.c

```diff
@@ -1282,6 +1283,37 @@ print_node(const char *pre_text, node_t * node, gboolean details)
     }
 }
 
+void
+pe__output_node(node_t * node, gboolean details, void * out)
```

Comment:
pe__output_node() and pe__output_resource() are internal API (like print_node() and print_resource()) so they don't need the void* trick, they can use pcmk__output_t directly. It would make sense for these two to go in pe_output.c.

Reply:
Fixed.